Library/Homebrew/os/mac/xcode.rb

```diff
@@ -317,15 +317,14 @@ def update_instructions
       sig { returns(String) }
       def latest_clang_version
         case MacOS.version
-        when "12"    then "1300.0.18.6"
-        when "11"    then "1205.0.22.9"
-        when "10.15" then "1200.0.32.29"
-        when "10.14" then "1100.0.33.17"
-        when "10.13" then "1000.10.44.2"
-        when "10.12" then "900.0.39.2"
-        when "10.11" then "800.0.42.1"
-        when "10.10" then "700.1.81"
-        else              "600.0.57"
+        when "11", "12" then "1300.0.29.3"
```

Comment:
```suggestion
        when "12", "11"  then "1300.0.29.3"
```

Totally optional. I really shouldn't care about this I'm sorry.

Reply:
I probably would have made the same review - don't worry! :)